src/lib/trusted-types.ts

```diff
@@ -0,0 +1,43 @@
+function getTrustedTypes() {
+  // tslint:disable-next-line
+  return (window as any).TrustedTypes;
+}
+
+const rules = {
+  createHTML(s: string): string {
+    return s;
+  },
+};
+let policy: typeof rules|undefined;
+
+/**
+ * Turns the value to trusted HTML. If the application uses Trusted Types the
+ * value is transformed into TrustedHTML, which can be assigned to execution
+ * sink. If the application doesn't use Trusted Types, the return value is the
+ * same as the argument.
+ */
+export function dangerouslyTurnToTrustedHTML(value: string): string {
+  const TrustedTypes = getTrustedTypes();
+  if (!policy && TrustedTypes !== undefined) {
+    policy = TrustedTypes.createPolicy('lit-html', rules);
```

Comment:
It's a little win for tests, though. If I initialize it in the module (where it will rely on Trusted Types being available in window) I may have problems with mocking TT in tests.

Reply:
Actually, I don't need this for polymer tests, because it uses a different stack than other frameworks where testing this was a bit harder. However it's good if we decide we want to make the policy name configurable (which for not isn't)